### PR TITLE
docs: add docs for BSC staking

### PR DESCRIPTION
**Pull request by forcodedancing**
add docs for BSC staking after BC fusion

### CODE REVIEW

File: docs/bcfusion/users/new-stake.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import Stake5 from '../../assets/bcfusion/user-stake5.png';
+import Stake6 from '../../assets/bcfusion/user-stake6.png';
+import Stake7 from '../../assets/bcfusion/user-stake7.png';
+import Stake8 from '../../assets/bcfusion/user-stake8.png';
+import Stake9 from '../../assets/bcfusion/user-stake9.png';
+import Stake10 from '../../assets/bcfusion/user-stake10.png';
+
+# New Stake Management
+
+You can new the BNB staking dApp to manage your staks easily.
```
Comment (zjubfd) [inline docs/bcfusion/users/new-stake.md:14]: `new the BNB staking dApp`?

Comment (zjubfd) [reply docs/bcfusion/users/new-stake.md:14]: typo `staks`

File: docs/bcfusion/users/new-stake.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+BNB Staking dApp on testnet: https://testnet-staking.bnbchain.org/en/bnb-staking
+
+BNB Staking dapp on mainnet: To be updated.
+
+## Connect Wallet
+
+<img src={Stake1} width="400"/>
+
+Firstly, you need to connect to a web3 wallets for mananging your stakes.
```
Comment (zjubfd) [inline docs/bcfusion/users/new-stake.md:24]: Please try to polish the word.

File: docs/bcfusion/users/new-stake.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+# Redelegate
+
+In `My Staking` page you can manage you existing delegations.
+
+By clicking `Redelegate` button, you can redelegate your stake to another validator.
+
+<img src={Stake6} width="400"/>
+
+In the popup window, you can choose the destination validator your want to stake to, as well as the amount you want to
```
Comment (zjubfd) [inline docs/bcfusion/users/new-stake.md:55]: please hightlight that the redelegate will charge 0.02% fee.

File: docs/bcfusion/users/stake-migration.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  confirm the address is correct to avoid funding lost.
+
+#### Setp 3: Sign the migrate transaction.
+
+Finally, you can sign the transaction and migration will be started.
+
+<img src={Migrate4} width="400"/>
+
+If the migration goes well, you will find the delegation in the new staking dApp. If the migration fails, the fund will
+be returned to your Beacon Chain, and you can check it in your free balance.
```
Comment (zjubfd) [inline docs/bcfusion/users/stake-migration.md:69]: `your free balance` is strange

File: docs/bcfusion/users/stake-migration.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+* The BSC delegator address: the address will be used as the owner for you to manage staking on the BSC. Please double
+  confirm the address is correct to avoid funding lost.
+
+#### Setp 3: Sign the migrate transaction.
+
+Finally, you can sign the transaction and migration will be started.
+
+<img src={Migrate4} width="400"/>
+
+If the migration goes well, you will find the delegation in the new staking dApp. If the migration fails, the fund will
```
Comment (zjubfd) [inline docs/bcfusion/users/stake-migration.md:68]: Please give more details about how to check the `delegation in the new staking dApp`, like put the new staking app link and connect the bsc wallet.

File: docs/bcfusion/users/stake-migration.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Step 2: Undelegate
+
+Then you can undelegate your delegations by click `Undelegate` button and send the transaction to the Beacon Chain.
+
+After the unbonding period (7 days in mainnet), the stake be returned to your Beacon Chain account.
+
+#### Step 3: Cross chain transfer
+
+You can use BEW or Trust to cross chain transfer your BNB from the Beacon Chain to the BSC.
```
Comment (zjubfd) [inline docs/bcfusion/users/stake-migration.md:91]: Please give more details here, as user may not know how to do cross chain transfer.

File: docs/bcfusion/validators/creation.md [hunk 7]
```diff
@@ -1,2 +1,137 @@
...
\ No newline at end of file
+import Validator1 from '../../assets/bcfusion/create-validator1.png';
+import Validator2 from '../../assets/bcfusion/create-validator2.png';
+import Validator3 from '../../assets/bcfusion/create-validator3.png';
+import Validator4 from '../../assets/bcfusion/create-validator4.png';
+
+# Validator Creation
+
+If you are migrating old validators created on
+the BNB Beacon Chain, please refer to [this document](migrations.md) for more information.
```
Comment (zjubfd) [inline docs/bcfusion/validators/creation.md:9]: I think `migrations.md` should be only suggested when the dapp is not available or the validator like to use cli.

I think `migrating old validators` can still use this dapp, just create the validator with proper `identity` and then migrate the delegation.

File: docs/bcfusion/validators/migrations.md [hunk 8]
```diff
@@ -1,2 +1,11 @@
-This page will guide validators to migrate their BSC validators.
-Please be patient for updates.
\ No newline at end of file
+# Validator Migration
+
+The [native staking](https://github.com/bnb-chain/BEPs/blob/master/BEPs/BEP294.md)
```
Comment (zjubfd) [inline docs/bcfusion/validators/migrations.md:3]: Please only suggest this method when the old validator would like to use cli and always guide them to use the dapp.

File: docs/bcfusion/validators/creation.md [hunk 9]
```diff
@@ -1,2 +1,137 @@
...
+
+After opening the dapp, you can click the `Become a Validator` button to create a new validator.
+
+The following information will be required in the `Create Validator` page.
+
+## Basic Info
+
+<img src={Validator1} width="400"/>
+
+- **Validator Name**: The name of your validator.
```
Comment (zjubfd) [inline docs/bcfusion/validators/creation.md:25]: Please describe the limitation rule of the name, like the length, character..

File: docs/bcfusion/validators/creation.md [hunk 10]
```diff
@@ -1,2 +1,137 @@
...
+An example proof
+is `0xaf762123d031984f5a7ae5d46b98208ca31293919570f51ae2f0a03069c5e8d6d47b775faba94d88dbbe591c51c537d718a743b9069e63b698ba1ae15d9f6bf7018684b0a860a46c812716117a59c364e841596c3f0a484ae40a1178130b76a5`.
+
+- `BSC_CHAIN_ID`: 56 for BSC mainnet, and 97 for BSC testnet.
+- `OPEATOR_ADDRESS`: The address of your account, which will be reconginzed as the operator of the new validator.
+- `VOTE_ADDRESS`: The vote address created in the last step.
+
+### Create indentity
+
+Please down load BC client binary here: https://github.com/bnb-chain/node/releases/tag/v0.10.18
```
Comment (zjubfd) [inline docs/bcfusion/validators/creation.md:77]: please hightlight that only the old validator who want to migrate need to do this, and no need for new BSC validators.

File: docs/bcfusion/users/new-stake.md (hunk 1 above)
Comment (forcodedancing) [reply docs/bcfusion/users/new-stake.md]: fixed

File: docs/bcfusion/users/new-stake.md (hunk 2 above)
Comment (forcodedancing) [reply docs/bcfusion/users/new-stake.md]: Fixed

File: docs/bcfusion/users/new-stake.md (hunk 3 above)
Comment (forcodedancing) [reply docs/bcfusion/users/new-stake.md]: Sure, added

File: docs/bcfusion/users/stake-migration.md (hunk 4 above)
Comment (forcodedancing) [reply docs/bcfusion/users/stake-migration.md]: Fixed

File: docs/bcfusion/users/stake-migration.md (hunk 5 above)
Comment (forcodedancing) [reply docs/bcfusion/users/stake-migration.md]: Added

File: docs/bcfusion/users/stake-migration.md (hunk 6 above)
Comment (forcodedancing) [reply docs/bcfusion/users/stake-migration.md]: Added

File: docs/bcfusion/validators/creation.md (hunk 7 above)
Comment (forcodedancing) [reply docs/bcfusion/validators/creation.md]: Sure. Updated the document.

File: docs/bcfusion/validators/creation.md (hunk 9 above)
Comment (forcodedancing) [reply docs/bcfusion/validators/creation.md]: Added

File: docs/bcfusion/validators/creation.md (hunk 10 above)
Comment (forcodedancing) [reply docs/bcfusion/validators/creation.md]: Sure. Added.

File: docs/bcfusion/users/new-stake.md [hunk 11]
```diff
@@ -9,70 +9,60 @@ import Stake8 from '../../assets/bcfusion/user-stake8.png';
...
-## Connect Wallet
+## Connecting Your Wallet
 
 <img src={Stake1} width="400"/>
 
-Firstly, you need to connect to a web3 wallets for mananging your stakes.
-`Trust` (mainnet only now) and `MetaMask` wallets are support, as well as other wallets which support `WalletConnect`.
-
-## Delegate
+To interact with the dApp, first connect your web3 wallet. Currently, `Trust` (mainnet only) and `MetaMask` are supported, along with any wallets compatible with `WalletConnect`.
```
Comment (zjubfd) [inline docs/bcfusion/users/new-stake.md:23]: `Trust` --> `Trustwallet`

File: docs/bcfusion/validators/creation.md [hunk 12]
```diff
@@ -3,51 +3,54 @@ import Validator2 from '../../assets/bcfusion/create-validator2.png';
 import Validator3 from '../../assets/bcfusion/create-validator3.png';
 import Validator4 from '../../assets/bcfusion/create-validator4.png';
 
-# Validator Creation
+# Validator Creation Guide
 
-If you are migrating old validators created on
-the BNB Beacon Chain, please refer to [this document](migrations.md) for more information.
+This guide outlines the process for creating a new validator on the BNB Smart Chain. If you're migrating validators
+from an older version, please consult [the migration document](migrations.md) for detailed instructions.
```
Comment (zjubfd) [inline docs/bcfusion/validators/creation.md:9]: If I am migrating validators from an older version, I can still refer to this page, dont I? The only thing I should take care is about the identiy.

File: docs/bcfusion/validators/creation.md [hunk 13]
```diff
@@ -68,13 +71,17 @@ geth_macos bls account generate-proof --chain-id ${BSC_CHAIN_ID} ${OPEATOR_ADDRE
...
+- `BSC_CHAIN_ID`: `56` for BSC mainnet, and `97` for BSC testnet.
 - `OPEATOR_ADDRESS`: The address of your account, which will be reconginzed as the operator of the new validator.
 - `VOTE_ADDRESS`: The vote address created in the last step.
 
 ### Create indentity
 
-Please down load BC client binary here: https://github.com/bnb-chain/node/releases/tag/v0.10.18
+Identity is used for assocaiting the new validator to the old validator created on the Beacon Chain, to facilitate
+delegators moving their stakes to the same validator operator when [migrations](../users/stake-migration.md).
+If you are creating a new validator without any need for such association, you can leave it empty.
```
Comment (zjubfd) [inline docs/bcfusion/validators/creation.md:82]: `If you are creating a new validator without any need for such association` ==> `If you never create a validator in this [page](https://www.bnbchain.org/en/staking)`

File: docs/bcfusion/validators/creation.md [hunk 14]
```diff
@@ -68,13 +71,17 @@ geth_macos bls account generate-proof --chain-id ${BSC_CHAIN_ID} ${OPEATOR_ADDRE
...
+
+Please down load BC client binary here: https://github.com/bnb-chain/node/releases/tag/v0.10.19
```
Comment (zjubfd) [inline docs/bcfusion/validators/creation.md:84]: typo `down load`...

File: docs/bcfusion/users/new-stake.md (hunk 11 above)
Comment (forcodedancing) [reply docs/bcfusion/users/new-stake.md]: fixed

File: docs/bcfusion/users/stake-migration.md [hunk 15]
```diff
@@ -88,8 +92,32 @@ After the unbonding period (7 days in mainnet), the stake be returned to your Be
 
 #### Step 3: Cross chain transfer
 
-You can use BEW or Trust to cross chain transfer your BNB from the Beacon Chain to the BSC.
+You can use BNB Chain Wallet (BEW) or Trust to cross chain transfer your BNB from the Beacon Chain to the BSC.
```
[the remark this answers is not in the document]
Comment (forcodedancing) [reply docs/bcfusion/users/stake-migration.md]: fixed

File: docs/bcfusion/validators/creation.md (hunk 12 above)
Comment (forcodedancing) [reply docs/bcfusion/validators/creation.md]: fxied

File: docs/bcfusion/validators/creation.md (hunk 13 above)
Comment (forcodedancing) [reply docs/bcfusion/validators/creation.md]: updated

File: docs/bcfusion/validators/creation.md (hunk 14 above)
Comment (forcodedancing) [reply docs/bcfusion/validators/creation.md]: fixed

**Review by zjubfd — APPROVED** [review]

**Pull request merged**